packages/frontend-2/components/dashboard/tutorials/Card.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <NuxtLink :to="webflowItem.url" target="_blank">
+  <NuxtLink :to="webflowItem.url" target="_blank" @click="trackClick">
```

Comment:
Were you able to test this to make sure the `to` doesn't interrupt the tracking call?

Reply:
Yeah because it opens in a new tab it doesn't intervere, original tab stays open and emits the event :)